### ISSUE TITLE
Expeditions needs gamer loot removed

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#17482]
dork and I are bikeshedding a better version that's less of a powergamer trap but in the interim remove the heals and the RPG and shit.

**Comment by EmoGarbage404** [comment]
Okay but like what would we replace them with? Guns make up 75% of the epic loot table.

**Comment by metalgearsloth** [comment]
For now just anything station-oriented like instrument crates or whatever.

We're bikeshedding removing the explicit mission structure and giving it more long-term progression.

**Comment by deltanedas** [comment]
me when epic mission giving a plushie crate (real)

also im gonna add some money to each tier since actually getting directly paid seems logical

**Comment by deltanedas** [comment]
rpg brutekits medkits swat removed can prob close

**Comment by Matz05** [comment]
So, what's the vision for expeditions? Currently, it's a straight combat challenge, on a role not actually allowed real combat equipment (which near-inevitably is issued [sanctioned or illicit] laser pistols by the QM to be able to do their jobs with a lower total-party-kill rate, and hoards anything else combat-relevant they can find, at least when I played while PKAs were removed), and it's going to earn strictly combat-irrelevant and other-job irrelevant fluff items?
I enjoyed when salvagers could take requests from other departments and be on the lookout to recover equipment and components of real value to the station on expeditions, it's sad to hear the dungeons being remapped to more or less static 'junk' scenery of no value.
Perhaps breaking up the 'combat check' and 'swarm of locusts loot everything' phases of expeditions could be done by breaking the current antagonistic relationship between Supply and Security and actually allowing trained, equipped soldiers onto away missions, Star Trek style (with a probably similar rate of fatalities, unfortunately).
Is the intent that more than just Supply goes on away missions? (Security officers as actual military help rather than commissars hunting for the disloyal, possibly medics or engineers for efficient stripping of the dungeon, etc.) Then having a staging area that's attached to, but not directly in the backrooms of Cargo, would be handy. People from other departments (especially Sec) entering Cargo is usually seen as an accusation of disloyalty, from what I've seen playing a lot of CT/Salv, and that tendency could be mitigated somewhat by having Salvage embark from a more 'neutral' area, while also making it easier to invite outsiders in without giving them access to whatever Cargo has lying around.
